data/strings/strings.txt

```diff
@@ -23454,3 +23454,21 @@
     vi = Dấu chân
     zh-Hans = 追踪
     zh-Hant = 追踪
+
+  [splash_subtitle]
+    comment = OpenStreetMap text on splash screen
+    tags = android,ios
+    en = Powered by OpenStreetMap
+    da = Drevet af OpenStreetMap
+    de = Angetrieben von OpenStreetMap
```

Comment:
"Angetrieben von" isn't the right meaning in this context. "Basierend auf" (based on) or "Basierend auf den Daten von" (based on the data from) would be better. Or "Powered by" since this term is also usual in the german region.

Reply:
Thank you! I've changed it to "Basierend auf OpenStreetMap" as the other one might be too long